policyengine_us/variables/gov/usda/summer_ebt/is_summer_ebt_eligible.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class is_summer_ebt_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for Summer EBT"
+    definition_period = YEAR
+    reference = (
+        "https://www.law.cornell.edu/uscode/text/42/1762#h_2"
+        "https://www.fns.usda.gov/summer/sunbucks",
+        "https://www.joinproviders.com/summer-ebt/",
+    )
+
+    def formula(person, period, parameters):
+        tax_unit = person.tax_unit
+        p = parameters(period).gov.usda.summer_ebt.eligibility
+        programs = add(tax_unit, period, p.categorical)
+        meal_plans = add(tax_unit, period, p.meal)
+        living_conditions = add(tax_unit, period, p.living_condition)
+        is_program_eligible = np.any(programs) & (
+            person("is_in_k12_school", period)
```

Comment:
Does the in highschool limitation apply across all of the eligibility criteria? If not could you add a comment explaining